src/web/app/pages-student/student-home-page/student-home-page.component.spec.ts

```diff
@@ -36,4 +36,60 @@ describe('StudentHomePageComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should snap with null fields', () => {
+    expect(fixture).toMatchSnapshot();
+  });
+
+  it('should snap with blank fields', () => {
+    const studentName: string = '';
+    const studentCourse: any = {
+      course: {},
+      feedbackSessions: [],
+    };
+    component.user = studentName;
+    component.courses = [studentCourse];
+    fixture.detectChanges();
+    expect(fixture).toMatchSnapshot();
+  });
+
+  it('should snap with populated fields', () => {
```

Comment:
Instead of "populated fields", you should take this chance to describe the test data. For example, this would be a student home page with one course with 2 feedback sessions.